src/qt/rpcconsole.cpp

```diff
@@ -436,10 +436,11 @@ void RPCConsole::clear()
     }
 
     // Set default style sheet
+    QFontDatabase::addApplicationFont(":/fonts/Inconsolata-Regular.ttf");
     ui->messagesWidget->document()->setDefaultStyleSheet(
                 "table { }"
-                "td.time { color: #808080; padding-top: 3px; } "
-                "td.message { font-family: Monospace; font-size: 12px; } "
+                "td.time { color: #808080; } "
```

Comment:
Does this need valign bottom too?

Reply:
I think it looks better with the valign bottom for the time because then it lines up with the message text...

Reply:
I think it might be inherited by the stylesheet but I will double check this.

Reply:
Looking at my screenshot, it looks like not, because in my screenshot, the bottom edge of the time is aligned with the bottom edge of the message text, but in your text it the bottom edge is higher. My guess is that the default is vertically centered rather than valign bottom...

Reply:
This is nitpicking I know...

Reply:
I have added the `valign: bottom;` to the message.  The problem is the icon size (as your is scaled), we need to play with that next.
Nitpicking is okay, it needs to look clean and professional.